common/static/common/js/spec/discussion/view/discussion_topic_menu_view_spec.js

```diff
@@ -45,7 +50,18 @@
                                     'sort_key': null,
                                     'is_cohorted': false,
                                     'id': 'c49f0dfb8fc94c9c8d9999cc95190c56'
+                                },
+                                'Very long category name': {
```

Comment:
Is "Very long category name" long enough to trigger truncation? It kicks in at 26, right?

```
$ > echo "Very long category name" | wc -m
      24
```